emloop/tests/cli/common_test.py

```diff
@@ -35,14 +52,25 @@ class SecondDummyHook(AbstractHook):
     pass
 
 
+class DummyConfigHook(AbstractHook):
+    """Dummy hook which changes config."""
+    def __init__(self, **kwargs):
+        self.kwargs = kwargs
+        super().__init__(**kwargs)
+        self.change_config(**kwargs)
+
+    def change_config(self, variables: List[str], **kwargs):
+        variables[0], variables[1] = variables[1], variables[0]
```

Comment:
I suggest moving this to the constructor. Also the `self.kwargs` are never used, no need to save them.